docker-compose.yml

```diff
@@ -12,10 +12,11 @@ services:
     shm_size: "256mb"
     build:
       context: .
-      dockerfile: docker/Dockerfile.dev
+      dockerfile: docker/Dockerfile
+      target: dev
     devices:
       - /dev/bus/usb:/dev/bus/usb
-      - /dev/dri:/dev/dri # for intel hwaccel, needs to be updated for your hardware
+      # - /dev/dri:/dev/dri # for intel hwaccel, needs to be updated for your hardware
```

Comment:
As it does not work for all users, I suggest to be commented out by default. BTW it does not work in my WSL2 environment.